spyder/plugins/help/widgets.py

```diff
@@ -17,7 +17,10 @@
 from qtpy import PYQT5, PYQT6
 from qtpy.QtCore import Qt, QUrl, Signal, Slot, QPoint
 from qtpy.QtGui import QColor
-from qtpy.QtWebEngineWidgets import WEBENGINE, QWebEnginePage
+try:
+    from qtpy.QtWebEngineWidgets import WEBENGINE
+except ImportError:
+    WEBENGINE = False
```

Comment:
```suggestion
```

We prefer try/except imports to not be in the middle of others, so I'll move this below.